test/QuadForm/LineOrbits.jl

```diff
@@ -0,0 +1,14 @@
+@testset "Random lines" begin
+  P = Hecke.enumerate_lines(GF(5), 20)
+  io = IOBuffer()
+  show(io, MIME"text/plain"(), P)
+  @test length(String(take!(io))) == 98
+  show(IOContext(io, :supercompact => true), P)
+  @test length(String(take!(io))) == 25
+  show(io, P)
+  @test length(String(take!(io))) == 33
```

Comment:
I don't understand this test :)

Reply:
I wanted to increase test coverage, but maybe I can remove the test on the length of the strings and just leave the test calling the show functions (kind of tautological tests but it is "easy" test coverage for the corresponding lines)